bci_main.py

```diff
@@ -36,7 +38,12 @@ def bci_main(parameters: dict, user: str, exp_type: int, mode: str) -> bool:
 
     # Initialize Save Folder
     save_folder = init_save_data_structure(
-        data_save_location, user, parameter_location, mode, exp_type)
+        data_save_location,
+        user,
+        parameter_location,
+        mode=mode,
+        experiment_type=exp_type,
+        experiment=experiment)
 
```

Comment:
bci_main is passing an extra parameter to init_save_data_structure which causes an error (`experiment`). Sorry I didn't catch this during the review.

Reply:
Or actually, it looks like maybe this needs to be `experiment_id`, rather than `experiment`?

Reply:
yeah - it should be experiment_id. I'll re-open a PR